base/cvd/cuttlefish/host/libs/web/android_build_api.cpp

```diff
@@ -365,7 +365,6 @@ Result<void> BuildApi::ArtifactToFile(const DeviceBuild& build,
                                       const std::string& artifact,
                                       const std::string& path) {
   const auto url = CF_EXPECT(GetArtifactDownloadUrl(build, artifact));
-  CF_EXPECT(CF_EXPECT(http_client->DownloadToFile(url, path)).HttpSuccess());
```

Comment:
nit: this is the only change needed for the fix, the `override`s are not related.

Reply:
Appreciate it. I'll reupload with just that change.